etcdctl/README.md

```diff
@@ -818,7 +818,7 @@ Get the status for all endpoints in the cluster associated with the default endp
 +------------------------+------------+
 |        ENDPOINT        |    HASH    |
 +------------------------+------------+
-| http://127.0.0.1:12379 | 1084519789 |
+| http://127.0.0.1:2379  | 1084519789 |
```

Comment:
Hi @opera443399 thanks for taking the time to create a PR. If you look at this example in context it seems that there are 3 etcd nodes on the same computer each on different ports. So while I see what you saying the example makes sense to me using ports **12379, 22379, 32379**.

Reply:
@hexfusion yes, I see.  
I'd like to use port `12379, 22379, 32379` too, but in context it seems like the `2379` is used by author, so I decide to replace `12379`(which affects 2 examples only) with `2379`

Reply:
@opera443399 thanks for the reply, read this doc over more. I see the reason for leaving this as 2379 as it is the default for etcdctl and thus does not require passing `--endpoints` flag. Although visually awkward your change is probably appropriate given the full context of the doc.